stable_baselines3/common/logger.py

```diff
@@ -99,7 +110,7 @@ def writekvs(self, kvs: Dict) -> None:
 
     @classmethod
     def _truncate(cls, string: str) -> str:
-        return string[:20] + '...' if len(string) > 23 else string
+        return string[:20] + '...' if len(string) > 30 else string
```

Comment:
I know this was not the case before, but I think this is a good time to have variables now instead of magic numbers ;)